mockserver-core/src/test/java/org/mockserver/persistence/ExpectationFileWatcherTest.java

```diff
@@ -383,16 +383,16 @@ public void shouldDetectModifiedInitialiserJsonOnAddWithGlob() throws Exception
                         .withBody("three second response")
                 )
             };
-            Files.write(mockserverInitializerThree.toPath(), expectationSerializer.serialize(expectionsThree).getBytes(StandardCharsets.UTF_8));
+            Files.write(mockserverInitializerThree.toPath(), expectationSerializer.serialize(expectationsThree).getBytes(StandardCharsets.UTF_8));
             long updatedFileTime = System.currentTimeMillis();
 
             Retries.tryWaitForSuccess(() -> assertThat(expectationsUpdatedCount.get(), equalTo(2)), 150, 100, MILLISECONDS);
             System.out.println("update processed in: " + (System.currentTimeMillis() - updatedFileTime) + "ms");
 
             // then
-            List<Expectation> expectations = requestMatchers.retrieveActiveExpectations(null);
-            assertThat(expectations.size(), equalTo(4));
-            assertThat(expectations, containsInAnyOrder(
+            List<Expectation> expectationsList = requestMatchers.retrieveActiveExpectations(null);
+            assertThat(expectationsList.size(), equalTo(4));
+            assertThat(expectationsList, containsInAnyOrder(
```

Comment:
As the types are different, I'm renaming this to `expectationsList`.

Again, it's possible to create a block scope, but the indentation required doesn't seem worth it.